### DISCUSSION TITLE
Does the plugin support External Event Dragging?

### DISCUSSION THREAD

**Discussion by celsoae** [saade/filament-fullcalendar#210] [Q&A]
I was trying to make it works for hours and  dont know if the plugin has support to it. ty in advance.

**Comment by krv** [comment]
Short answer: no
See relevant discussion here: https://github.com/saade/filament-fullcalendar/discussions/190
Feature request for v4 here: https://github.com/saade/filament-fullcalendar/issues/201#issuecomment-2254096294
